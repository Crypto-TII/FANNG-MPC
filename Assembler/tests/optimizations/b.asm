ldi c0, 3
ldsi s0, 3
crash
# these statements are unreachable
# FIXME: report a better error
clear_memory
clear_registers
