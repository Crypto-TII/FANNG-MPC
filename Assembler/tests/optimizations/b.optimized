crash
clear_memory
clear_registers
