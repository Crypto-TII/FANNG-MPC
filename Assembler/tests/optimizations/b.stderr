warning: Write to register is never read
 --> tests/optimizations/b.asm:2:1
  |
2 | ldsi s0, 3
  | ---------- register 0 written to but will never be read from again
  |
warning: Write to register is never read
 --> tests/optimizations/b.asm:1:1
  |
1 | ldi c0, 3
  | --------- register 0 written to but will never be read from again
  |